src/main/kotlin/net/maizegenetics/phgv2/brapi/utilities/LocalDateTimeAsStringSerializer.kt

```diff
@@ -0,0 +1,27 @@
+package utilities
+
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.descriptors.PrimitiveKind
+import kotlinx.serialization.descriptors.PrimitiveSerialDescriptor
+import kotlinx.serialization.descriptors.SerialDescriptor
+import kotlinx.serialization.encoding.Decoder
+import kotlinx.serialization.encoding.Encoder
+import java.time.LocalDateTime
+
+/**
+ * A LocalDateTimeAsStringSerializer is a custom serializer for LocalDateTime objects,
+ * as it's not supported by kotlinx.serialization by default.
+ */
+object LocalDateTimeAsStringSerializer : KSerializer<LocalDateTime> {
```

Comment:
You might want to add some simple unit tests to make sure this is working correctly.  Should be very simple to verify it all works.